frame/transaction-payment/src/lib.rs

```diff
@@ -194,13 +353,14 @@ impl<T: Trait> Module<T> where
 		if pays_fee == Pays::Yes {
 			let len = <BalanceOf<T>>::from(len);
 			let per_byte = T::TransactionByteFee::get();
-			let len_fee = per_byte.saturating_mul(len);
-			let unadjusted_weight_fee = Self::weight_to_fee(weight);
 
+			let unadjusted_len_fee = per_byte.saturating_mul(len);
+			let unadjusted_weight_fee = Self::weight_to_fee(weight);
 			// the adjustable part of the fee
-			let adjustable_fee = len_fee.saturating_add(unadjusted_weight_fee);
-			let targeted_fee_adjustment = NextFeeMultiplier::get();
-			let adjusted_fee = targeted_fee_adjustment.saturating_mul_acc_int(adjustable_fee);
+			let adjustable_fee = unadjusted_len_fee.saturating_add(unadjusted_weight_fee);
```

Comment:
Didn't you say that you change it so that the len_fee is not adjusted by the multiplier?

Reply:
I posted that 20 mins ago :D not pushed yet :D

Reply:
Now you can review again.

Reply:
Looks good.